app/assets/stylesheets/panel-grey.css

```diff
@@ -0,0 +1,4 @@
+.panel-grey{
+    color: #0b0c0c;
+    background-color: #f3f2f1;
```

Comment:
Should we be using the [colour palette](https://design-system.service.gov.uk/styles/colour/#colour-palette) here, so that we say ` background-color: govuk-colour("light-grey");` instead of `background-color: #f3f2f1;` etc? Or is that overkill?